src/NSwag.Generation.AspNetCore/AspNetCoreOpenApiDocumentGenerator.cs

```diff
@@ -80,7 +80,7 @@ dynamic GetJsonOptionsWithReflection(IServiceProvider sp)
                 }
             }
 
-#if NET5_0 || NETCOREAPP3_1
+#if NETCOREAPP3_1_OR_GREATER
```

Comment:
Is NETCOREAPP3_1_OR_GREATER really defined?

Reply:
Apparently it's Only